README.md

```diff
@@ -731,8 +878,8 @@ The options to the configuration file are the same the [API options](#options).
 JSON:
 
 ```json
-// .prettierrc
```

Comment:
We don't want this comment to move. Perhaps `<!-- prettier-ignore -->` this too.